OtripleS.Web.Api.Tests.Unit/Services/StudentExams/StudentExamServiceTests.cs

```diff
@@ -41,9 +41,6 @@ public StudentExamServiceTests()
         private static DateTimeOffset GetRandomDateTime() =>
             new DateTimeRange(earliestDate: new DateTime()).GetValue();
 
-        private IQueryable<StudentExam> CreateRandomStudentGuardians() =>
-           CreateStudentExamFiller(DateTimeOffset.UtcNow).Create(GetRandomNumber()).AsQueryable();
-
         private StudentExam CreateRandomStudentExam(DateTimeOffset dateTime) =>
```

Comment:
Why are we deleting this?

Reply:
StudentExamServiceTests has CreateRandomStudentExam and this method is irrelevant  CreateRandomStudentGuardians which was causing build error in Azure pipelines.